packages/core/src/components/grading-instructions/GradingInstructions.tsx

```diff
@@ -98,26 +105,37 @@ const GradingInstructions: React.FunctionComponent<CommonExamProps> = ({ doc })
   useEffect(scrollToHash, [])
 
   return (
-    <I18nextProvider i18n={i18n}>
-      <main className="e-exam e-grading-instructions" lang={subjectLanguage}>
-        <React.StrictMode />
-        <SectionElement aria-labelledby={examTitleId}>
-          <DocumentTitle id={examTitleId}>
-            <GradingInstructionsPageTitle /> {examTitle && renderChildNodes(examTitle)}
-          </DocumentTitle>
-          {date && (
-            <p>
-              <strong>{dateTimeFormatter.format(date)}</strong>
-            </p>
-          )}
-          {examGradingInstruction && (
-            <ExamGradingInstruction {...{ element: examGradingInstruction, renderChildNodes }} />
-          )}
-          {tableOfContents && <TableOfContents {...{ element: tableOfContents, renderChildNodes }} />}
-        </SectionElement>
-        {renderChildNodes(root)}
-      </main>
-    </I18nextProvider>
+    <AnnotationProvider
```

Comment:
Vähän harmittaa että exam-maker kohtainen AnnotationProvider on näin näkyvä tässä ja exam.tsx:ssä mutta ei sille taida mahtaa mitään.